net-libs/libsrtp/libsrtp-1.5.4.ebuild

```diff
@@ -0,0 +1,76 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit autotools
+
+DESCRIPTION="Open-source implementation of the Secure Real-time Transport Protocol (SRTP)"
+HOMEPAGE="https://github.com/cisco/libsrtp"
+SRC_URI="https://github.com/cisco/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0/1"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 -sparc ~x86 ~x86-fbsd ~ppc-macos ~x64-macos ~x86-macos"
+IUSE="aesicm console debug doc libressl openssl static-libs syslog test"
+
+DEPEND="
+	openssl? ( dev-libs/openssl:0= )
+	libressl? ( dev-libs/libressl:0= )
+"
```

Comment:
@SoapGentoo 
That page only shows the
ssl? ( !libressl? ( dev-libs/openssl )
         libressl? ( dev-libs/libressl ) )
construct

Anyway, my understanding from that page was that if a user had USE="libressl ssl" set globally, it should enable working libressl support on all packages that support an openssl-like backend.

Sorry, I made a mistake. I'm changing it back to what you have.